workflows/taco.nf

```diff
@@ -7,6 +7,7 @@
 include { paramsSummaryMap                       } from 'plugin/nf-schema'
 include { softwareVersionsToYAML                 } from '../subworkflows/nf-core/utils_nfcore_pipeline/main.nf'
 include { paramsSummaryMultiqc                   } from '../subworkflows/nf-core/utils_nfcore_pipeline/main.nf'
+include { completionEmail                        } from '../subworkflows/nf-core/utils_nfcore_pipeline/main.nf'
```

Comment:
Why did you add this here and not in:

`subworkflows/local/utils_nfcore_taco_pipeline/main.nf` ?